content/blog/nats-connect-live-2020.md

```diff
@@ -0,0 +1,126 @@
+</a>
+  </div>
+  <div class="grid-item">
+<img src="/img/blog/nats-connect-live-2020/resgate-slide.png" >
+</div>
+</div>
+
+#### 3. NATS and Augmented Reality
+<div class="grid-container">
+ <div class="grid-item">
+This was a fascinating talk from Ron Britvich on how NATS is enabling real-time events at a massive scale and updating for local frame of reference in the Unity game engine. <br><br>
+ <a href="https://www.slideshare.net/nats_io/nats-connect-live-nats-augmented-reality" target="_blank">
+   <img src="/img/social/slideshare.png" height="65" width="90">
+   </a><br>
+ <a href="https://www.youtube.com/watch?v=7AGY9iiOgmk&list=PLose4NjLOQvwMOG-WAw62vyrGsPi5LUe7&index=3" alt="Unity NATS Subjects" target="_blank">
+ <img src="/img/social/youtube.png" height="60" width="85">
+ </a>
+  </div>
+  <div class="grid-item">
+<img src="/img/blog/nats-connect-live-2020/unity-slide.png" >
+</div>
+</div>
+
+#### 4. Cloud Native WebAssembly
+<div class="grid-container">
+ <div class="grid-item">
+I am a big fan of WASM’s potential and we believe it will play a large role in the NATS ecosystem in the near future. Kevin Hoffman is always pushing the boundaries and his project is no different! He discusses the project and how NATS plays a role.
+ <br>
+ <br>
+ <a href="https://slides.com/autodidaddict/nats-connect-cloud-native-webassembly#/" target="_blank">
+   <img src="/img/social/slides_logo.png" height="50" width="75">
+   </a><br>
+ <a href="https://www.youtube.com/watch?v=7AGY9iiOgmk" alt="Unity NATS Subjects" target="_blank">
+ <img src="/img/social/youtube.png" height="60" width="85">
+ </a>
+  </div>
+  <div class="grid-item">
+<img src="/img/blog/nats-connect-live-2020/wasm-slide.png" >
+</div>
+</div>
+
+#### 4. SwimOS and NATS
+<div class="grid-container">
+ <div class="grid-item">
+Enabling edge systems has become a sweet spot for NATS. Enabling remote edge environments to work autonomously but also communicate with more centralized resources is a dominant pattern for distributed data processing. Simon Crosby walks us through what this will really look like in the near future with the <a href="https://swim.ai">Swim.ai</a> platform and why the team think NATS is such a good enabler for these types of platforms. 
+<a href="https://www.slideshare.net/nats_io/nats-connect-live-swimos-nats" target="_blank">
+   <br><br /><img src="/img/social/slideshare.png" height="65" width="90">
+   </a><br>
+ <a href="https://www.youtube.com/watch?v=J56wrBDG01k" alt="Unity NATS Subjects" target="_blank">
+ <img src="/img/social/youtube.png" height="60" width="85">
+ </a>
+  </div>
+  <div class="grid-item">
+<img src="/img/blog/nats-connect-live-2020/swim-slide.png" >
+</div>
+</div>
+<br /><br />
+All of the talks are worth checking out, and we look forward to having these events more often. Feel free to reach me directly with any feedback or questions.
+
+Contact NATS [info@nats.io](mailto:info@nats.io).
```

Comment:
That is not me directly, so either change sentence or put my email derek@nats.io

Reply:
I didn't know if you wanted your email or what. I can change to you.